### PR TITLE
enable neutron lbaas dashboard

### PR DESCRIPTION
**Pull request by zfeldstein**

### CODE REVIEW

**Comment by zfeldstein** [comment]
retest

**Comment by j2sol** [comment]
retest

File: roles/horizon/tasks/main.yml [hunk 1]
```diff
@@ -93,8 +93,13 @@
     - { name: img/favicon.ico, url: "{{ horizon.favicon_url|default('') }}" }
   when: (horizon.logo_url is defined and horizon.favicon_url is defined)
 
+- name: Enable/Disable lbaas dashboard
+  file:
+    src: "{{ horizon.horizon_lib_dir }}/lib/python2.7/site-packages/neutron_lbaas_dashboard/enabled/_1481_project_ng_loadbalancersv2_panel.py"
+    path: "{{ horizon.horizon_lib_dir }}/lib/python2.7/site-packages/openstack_dashboard/local/enabled/_1481_project_ng_loadbalancersv2_panel.py"
+    state: "{{ (neutron.lbaas.enable | bool) | ternary('link', 'absent') }}"
```
Comment (j2sol) [inline roles/horizon/tasks/main.yml]: That's a neat trick.

**Comment by nirajdp76** [comment]
:+1:

**Pull request merged**